### ISSUE TITLE
Building in Windows Standalone

### ISSUE THREAD

**Issue by QudamahAMBS** [reinterpretcat/utymap#131]
Many thanks for the amazing work in this project.
I am wondering if someone could help me with the following issue:
I am using the release version of the code and want to publish it in a standalone (windows) execution file.
The project is running with no issues within the Unity Editor but when I tried to build and run it in a standalone windows environment, it built successfully but did not work as expected (the third person is loaded but none of the maps' parts is loaded.
Anything associated with the plugins dll files are not working.

Any idea or suggestion to solve this issue will be appreciated.

Kindest regards
Qudamah

**Comment by reinterpretcat** [comment]
Thanks for feedback!

First thing I would check is that all dlls (e.g. zlib, UtyDepend, UtyRx, etc.) are also included with release build.

Next thing: adding extra logging to catch the failure.

**Comment by QudamahAMBS** [comment]
Dear reinterpretcat,
Thanks for your valuable feedback... 
I took your advice and I added extra logging as all the dlls files are already included. Then I noticed few files and folders are missing within the local application data folder (user\AppData\LocalLow\projectname\Demo\ ) including: config and mapcss  folders from the StreamingAssets
Now it is working fine.

Many thanks again for your advice.
Regards
